apps/meteor/client/components/GenericMenu/GenericMenu.tsx

```diff
@@ -46,7 +46,7 @@ const GenericMenu = ({ title, icon = 'menu', disabled, onAction, ...props }: Gen
 	const isMenuEmpty = !(sections && sections.length > 0) && !(items && items.length > 0);
 
 	if (isMenuEmpty || disabled) {
-		return <IconButton small icon={icon} disabled />;
+		return <IconButton small icon={icon} title={t.has(title) ? t(title) : title} disabled />;
```

Comment:
this change isn't necessary at all

Reply:
I think the ternary is not necessary indeed...

Reply:
As discussed previously, disabled buttons don't need a title on it. Removing title prop!